### ISSUE TITLE
Can't sign in

### ISSUE THREAD

**Issue by jplatte** [microsoft/vscode-pull-request-github#2729]
- Extension version: 0.26.0
- VSCode Version: 1.56.1
- OS: arch linux

Steps to Reproduce:

1. Install extension
2. Click sign in button
3. Click Continue on the "Authorize Visual Studio Code to access GitHub" page that opens
4. Get error message

![Screenshot_2021-05-12_200122](https://user-images.githubusercontent.com/951129/118022650-da8d7980-b35c-11eb-88ce-aa9a3dc96e6f.png)

**Comment by alexr00** [comment]
I'm not able to repro this issue. I'm guessing is an compatibility issue with arch. In VS Code, can you open the Output panel and share the content of the GitHub Authentication output after this issue occurs?

**Comment by jplatte** [comment]
It just says

```
[Info  - 12:43:52.996] Logging in...
```

Until I cancel it:

```
[Error  - 12:44:19.770] Cancelled
```

VSCode never gets a reply from the browser since the auth process errors there. (I've tried with both Firefox and Chromium by the way)

**Comment by alexr00** [comment]
Hmm. @TylerLeonhardt do you have any insights here?

**Comment by TylerLeonhardt** [comment]
@alexr00 I believe this is a duplicate of https://github.com/microsoft/vscode-pull-request-github/issues/1221

**Comment by jplatte** [comment]
Hm, that issue is about an "Error 801" where I'm getting "Forbidden", but I'll watch it and see whether it works for me too when that issue is resolved.

**Comment by TylerLeonhardt** [comment]
Hmm could be a different issue then... Are you using firefox? @jplatte

**Comment by jplatte** [comment]
Yeah but as noted above I tried Chromium too (not by copying the link after it had already failed but by setting it as my default browser temporarily).

**Comment by TylerLeonhardt** [comment]
@jplatte which desktop environment do you use?

**Comment by jplatte** [comment]
I'm using [sway](https://github.com/swaywm/sway).

**Comment by jbuchmann-castor** [comment]
I'm having this same problem (getting the Forbidden screen after clicking the Continue button) on both Firefox and Chromium. I am trying with version 1.59 of vscodium and 0.29.0 of the extension. My system details:
```
$ uname -a
Linux LAT-14 5.11.0-31-generic #33-Ubuntu SMP Wed Aug 11 13:19:04 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 21.04
Release:	21.04
Codename:	hirsute

$ code --version
1.59.0
379476f0e13988d90fab105c5c19e7abc8b1dea8
x64

$ gnome-shell --version
GNOME Shell 3.38.4
```
